dashlane/vault.go

```diff
@@ -47,14 +48,21 @@ type Vault struct {
 	Passwords []PasswordEntry `xml:"KWAuthentifiant>KWDataItem"`
 }
 
-func ParseVault(data, password string) (Vault, error) {
+func ParseVault(data, password string) (*Vault, error) {
+	vault := new(Vault)
 	decrypted, err := DecryptVault(data, password)
 	if err != nil {
-		return err
+		return vault, err
 	}
 
 	// parse the XML
+ err = xml.Unmarshal([]byte(decrypted), &vault)
```

Comment:
You should run `go fmt` to respect go standards :)

Reply:
I ran go fmt on the project